data_steward/tools/create_ehr_and_rdr_snapshot.sh

```diff
@@ -46,7 +41,7 @@ while true; do
   esac
 done
 
-if [[ -z "${key_file}" ]] || [[ -z "${ehr_dataset}" ]] || [[ -z "${rdr_dataset}" ]] || [[ -z "${validation_dataset}" ]] || [[ -z "${dataset_release_tag}" ]] ; then
+if [[ -z "${key_file}" ]] || [[ -z "${ehr_dataset}" ]] || [[ -z "${rdr_dataset}" ]] || [[ -z "${dataset_release_tag}" ]] ; then
   echo "Specify the key file location and ehr_dataset ID, rdr_dataset ID, validation_dataset ID and Dataset release tag . $USAGE"
```

Comment:
This mentions the validation dataset; might be easier to stay consistent by just echoing usage:
```suggestion
  echo "${USAGE}"
```

Reply:
Nice catch.  I missed that reference.